utils/create_dependency_mapping.py

```diff
@@ -54,7 +59,7 @@ def extract_classes_and_imports(file_path):
     for node in ast.walk(tree):
         if isinstance(node, (ast.Import, ast.ImportFrom)):
             module = node.module if isinstance(node, ast.ImportFrom) else None
-            if module and (".modeling_" in module):
+            if module and (".modeling_" in module or "transformers.models" in module):
```

Comment:
This was changed in a recent PR, but seems to have broken modular for generating file other than modeling

Reply:
yep, let's be more restrictive I think I was trying to avoid imports that are not from a modeling code directly, like models.auto etc. Can you make sure the order is still correct?

Reply:
The order was off for `instruct_blib_video`, because `llava_next` was listed twice as a dep. I added a check right after this line to avoid this and now the order is the same as it was